Router/RedirectRouter.php

```diff
@@ -135,6 +121,18 @@ public function getContext()
      */
     public function getRouteCollection()
     {
+        if($this->redirects === false) {
```

Comment:
You don't have to set `$this->redirects` beforehand. Simply check `if ($this->redirects === null)`. Please don't forget the space between `if` and `(`.